es/balance.go

```diff
@@ -23,87 +23,38 @@ type Balance struct {
 	ID        string        `json:"-"`
 	AccountID string        `json:"account_id"`
 	Balance   string        `json:"balance"`
+	Amount    string        `json:"amount"`
 	CreatedAt time.Time     `json:"created_at"`
 	Source    BalanceSource `json:"source"`
 	Asset     Asset         `json:"asset"`
 }
 
 // NewBalanceFromAccountEntry creates Balance from AccountEntry
-func NewBalanceFromAccountEntry(a xdr.AccountEntry, now time.Time, id string, source BalanceSource) *Balance {
+func NewBalanceFromAccountEntry(a xdr.AccountEntry, amt xdr.Int64, now time.Time, id string, source BalanceSource) *Balance {
 	return &Balance{
 		ID:        id,
 		AccountID: a.AccountId.Address(),
 		Balance:   amount.String(a.Balance),
+		Amount:    amount.String(amt),
```

Comment:
Not sure about the naming 🤔 It seems ambiguous.

Maybe we should rename the whole `Balance` struct to something like `BalanceState` with fields `Value` (or `BalanceValue`) and `Diff` instead of `Balance` and `Amount` correspondingly?

Reply:
Understand your concern. I believe, it would be better to keep base structure name as is, but Value and Diff sounds better, will fix.